### ISSUE TITLE
Draw Axes on the Board

### ISSUE THREAD

**Issue by stojadin2701** [catsystems/cats-hardware#2]
Knowing the reference axes and how the board was placed in the rocket can help when analyzing the flight data.

@jbinz if we record the calibration info in the "flight stats" file, can we reconstruct in which orientation the board was mounted? This info shouldn't change during flight, only when transitioning from Moving -> Ready?

**Comment by jbinz** [comment]
Yes, true. This is a feature I wanted to add anyway, to have this saved in the stats file

**Comment by jbinz** [comment]
But one cannot fully get the orientation. Just the up direction.
